components/server/src/express/ws-handler.ts

```diff
@@ -89,7 +89,12 @@ export class WsExpressHandler implements Disposable {
                 .finally(() => {
                     const pathname = request.url ? url.parse(request.url).pathname : undefined;
                     const method = request.method || "UNKNOWN";
-                    increaseHttpRequestCounter(method, pathname || "unkown-websocket", request.statusCode || 0);
+                    increaseHttpRequestCounter(
+                        method,
```

Comment:
Hm. I wonder whether we might be happier with a separate counter, just for the httpVersion.

Reply:
Happier how?

It seemed "natural" to do as an additional label on the existing metric, because then we can slice and dice using it's other labels, too.

Reply:
You are probably right. Let's keep it as-is. :+1: